py3status/modules/bluetooth.py

```diff
@@ -55,47 +52,28 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     device_separator = '|'
-    format = '{name}'
-    format_no_conn = 'OFF'
-    format_no_conn_prefix = 'BT: '
-    format_prefix = 'BT: '
+    format = 'BT[: {format_device}]'
```

Comment:
> format = 'BT[: {format_device}]' 

I can live with this

> Also with new dbus stuffs, you have to run bluetooth service. 

could you explain a little more.  This also sounds like maybe it should be a new PR once this one is done